File: output/CodeGen/avatar/Java/Python/atcoder_AGC002_A.py
def main(args):
    scanner = Scanner(System.in)
    a = scanner.nextint()
    b = scanner.nextint()
    if a > 0:
        print("Positive")
    elif b >= 0:
        print("Zero")
    elif (b - a + 1) % 2 == 0:
        print("Positive")
    else:
        print("Negative")

File: output/CodeGen/avatar/Python/Java_repair/atcoder_AGC002_A.java
import java.util.*;
import java.io.*;

public class atcoder_AGC002_A {
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        String s;
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        if (a > 0) {
            System.out.println("Positive");
        } else if (a <= 0 && b >= 0) {
            System.out.println("Zero");
        } else {
            if ((a + b) % 2 == 0) {
                System.out.println("Negative");
            } else {
                System.out.println("Positive");
            }
        }
    }
}